libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -657,6 +666,12 @@ namespace eosio { namespace testing {
       bool                     skip_validate = false;
    };
 
+   class validating_tester_no_disable_deferrd_trx : public validating_tester {
```

Comment:
Typo in `deferrd`. Should be called `validating_tester_no_disable_deferred_trx`.